index.html

```diff
@@ -828,134 +605,23 @@ <h3>Minimizing information exposure</h3>
-             </li>
-             <li>
-              Speeds between these values are linearly interpolated.
-             </li>
-           </ul>
-        </p>
-      </section>
-      <section>
-        <h4>Aggregating CPU frequency</h4>
-        <p class="issue">
-          TODO: Aggregating is an average of the current speed across all cores. No
-          aggregation over a time window. Proposal for aggregating frequencies across
-          systems with heterogeneous CPU cores
-        </p>
-      </section>
-      <section>
-        <h4>Quantizing values (a.k.a. Bucketing)</h4>
-        <p>
-          Quantizing the aggregated CPU utilization and frequency reduces the amount of
-          information exposed by the API.
-        </p>
-        <p>
-          Having applications designate the quantization ranges (buckets) reduces the
-          quantization resolution that user agents must support in order to enable the
-          decisions used in a multitude of applications.
-        </p>
-        <p>
-          Applications communicate their desired quantization scheme by passing in a list
-          of thresholds. For example, the thresholds list `[0.5, 0.75, 0.9]` defines a
-          4-bucket scheme, where the buckets cover the value ranges `0`-`0.5`, `0.5`-`0.75`,
-          `0.75`-`0.9`, and `0.9`-`1.0`. We propose representing a bucket using the middle
-          value in its range.
-        </p>
-        <p class="example">
-          Suppose an application used the threshold list above, and the user
-          agent measured a CPU utilization of `0.87`. This would fall under the `0.75`-`0.9`
-          bucket, and would be reported as `0.825` (the average of `0.75` and `0.9`).
-        </p>
-        <p>
-          We recommend that user agents allow at most 5 buckets (4 thresholds) for
-          CPU utilization, and 2 buckets (1 threshold) for CPU speed.
-        </p>
-      </section>
       <section>
         <h4>Rate-limiting change notifications</h4>
         <p>
-          We propose exposing the quantized CPU utilization and frequency via
+          We propose exposing the pressure state via
           rate-limited change notifications. This aims to remove the ability to observe
-          the precise time when a value transitions between two buckets.
+          the precise time when a value transitions between two states.
         </p>
         <p>
-          More precisely, once the compute pressure observer is installed, it will be
-          called once with initial quantized values, and then be called when the quantized
-          values change. The subsequent calls will be rate-limited. When the callback is
-          called, the most recent quantized value is reported.
+          More precisely, once the compute pressure observer is activated, it will be
+          called once with initial values, and then be called when the values change.
+          The subsequent calls will be rate-limited. When the callback is
```

Comment:
Can you add a note maybe explaining how user agents could implement this rate limitation?

Reply:
added

Reply:
It's below :-) 

        <p>
          Rate limiting can be implemented in the user agent, but it might also be
          possible to simply change the polling frequency of the underlying hardware
          counters, if not accessed via a higher level framework.
        </p>

Reply:
Got it. Thanks!